src/components/ImportExport.vue

```diff
@@ -169,10 +176,10 @@ export default {
           const ss = obj.selectedSubjects.map((s) => {
             // make sure it has everything, if not fill in from subjectsIndex or genericCourses
             let subject;
-            if (this.subjectsIndex[s.id] !== undefined) {
-              subject = this.subjects[this.subjectsIndex[s.id]];
-            } else if (this.genericIndex[s.id] !== undefined) {
-              subject = this.genericCourses[this.genericIndex[s.id]];
+            if (this.$store.state.subjectsIndex[s.id] !== undefined) {
+              subject = this.$store.state.subjectsInfo[this.$store.state.subjectsIndex[s.id]];
+            } else if (this.$store.state.genericIndex[s.id] !== undefined) {
+              subject = this.$store.state.genericCourses[this.$store.state.genericIndex[s.id]];
             }
 
```

Comment:
Line 203: `this.$emit('add-road', this.roadtitle, obj.coursesOfStudy, ss, obj.progressOverrides);` 

Is there a reason adding a road is not done by the $store?

Reply:
Another thing I missed, will send this up with the other fixes.

Reply:
Just kidding, I remember now. `addRoad` needs access to the `Auth` `ref`, which we can't retrieve from the `Store`, hence why I left it as is. It also doesn't make much sense to move `newRoads` out into the `Store` just for this one case.